tests/unit/Core/Content/Cookie/Service/CookieProviderTest.php

```diff
@@ -85,24 +85,24 @@ public function testGetCookieGroupsWithTranslation(): void
             $translator,
             ['name' => 'test-session-name-']
         )
-        )->getCookieGroups(Generator::generateSalesChannelContext(), true);
+        )->getCookieGroups(Generator::generateSalesChannelContext());
 
         static::assertCount(3, $cookieGroups);
         $group = $cookieGroups->get('cookie.group.test');
         static::assertInstanceOf(CookieGroup::class, $group);
-        static::assertSame('cookie.group.test', $group->snippetKeyName);
-        static::assertSame('Translated: cookie.group.test', $group->translatedName);
-        static::assertSame('cookie.group.test.description', $group->snippetKeyDescription);
-        static::assertSame('Translated: cookie.group.test.description', $group->translatedDescription);
+        static::assertSame('Translated: cookie.group.test', $group->name);
+        static::assertSame('Translated: cookie.group.test.description', $group->description);
+        $groupNameProp = new \ReflectionProperty(CookieGroup::class, 'snippetKeyName');
```

Comment:
can you use `\Shopware\Core\Framework\Test\TestCaseHelper\ReflectionHelper::getProperty` ?